src/main/resources/assets/music_control/lang/en_us.json

```diff
@@ -99,19 +102,39 @@
     "minecraft:music/menu/menu2": "C418 - Moog City 2",
     "minecraft:music/menu/menu3": "C418 - Beginning 2",
     "minecraft:music/menu/menu4": "C418 - Floating Trees",
+
+    "minecraft:music/game/ancestry": "Lena Raine - Ancestry",
+    "minecraft:music/game/an_ordinary_day": "Kumi Tanioka - An Ordinary Day",
+    "minecraft:music/game/comforting_memories": "Kumi Tanioka - Comforting Memories",
+    "minecraft:music/game/floating_dream": "Kumi Tanioka - Floating Dream",
+    "minecraft:music/game/infinite_amethyst": "Lena Raine - Infinite Amethyst",
+    "minecraft:music/game/left_to_bloom": "Lena Raine - Left to Bloom",
+    "minecraft:music/game/one_more_day": "Lena Raine - One More Day",
+    "minecraft:music/game/stand_tall": "Lena Raine - Stand Tall",
+    "minecraft:music/game/wending": "Lena Raine - Wending",
+
```

Comment:
Remove spaces between `game` music, it's already a long enough file.  
Same comment for `game/nether` music.